functions/startup/routes.js

```diff
@@ -20,5 +22,7 @@ module.exports = function(app) {
     app.use('/api/users', users);
     app.use('/api/roles', roles);
     app.use('/api/permissions', permissions);
+    app.use('/api/audits',audits);
```

Comment:
if audit file is not added, can you please remove audit. Srikant also has created PR for audit. It will create conflicts otherwise.